config/routes.rb

```diff
@@ -209,6 +209,10 @@
           resources :materials, path: 'files'
         end
       end
+
+      resources :course_users, only: [] do
+        resources :experience_points_records, only: [:new, :create, :index]
```

Comment:
Actually now I'm curious. This page is scoped under a user, so it cannot be used without referencing a particular user. What are you trying to do?

I don't imagine any circumstance where a TA will go into a user's page one by one and grant EXP.